src/Stylist/Theme/Stylist.php

```diff
@@ -125,9 +124,7 @@ public function activate($theme)
 
         $this->activeTheme = $theme;
 
-        $this->activateFinderPaths($theme);
-
-        Log::info("Using theme [{$theme->getName()}]");
+        $this->activateFinderPaths($theme;
```

Comment:
There's no issue - the tests would have broken if it were true. I've checked the file now and it's fine :)